clang/lib/CodeGen/CodeGenAction.cpp

```diff
@@ -21,6 +21,7 @@
 #include "clang/Basic/LangStandard.h"
 #include "clang/Basic/SourceManager.h"
 #include "clang/Basic/TargetInfo.h"
+#include "clang/CIRFrontendAction/CIRGenAction.h"
```

Comment:
You need to isolate this addition under cmake generated CIR macro, because we can build clang without CIR support, there are examples around.